main.js

```diff
@@ -0,0 +1,37 @@
+var car = document.querySelector('img');
+
+// Rotate Down
+function rotateDown(event) {
+  if (event.code !== 'ArrowDown') {
+    return;
+  }
+  car.className = 'car transition down';
+}
+window.addEventListener('keydown', rotateDown);
+
+// Rotate Up
+function rotateUp(event) {
+  if (event.code !== 'ArrowUp') {
+    return;
+  }
+  car.className = 'car transition up';
+}
+window.addEventListener('keydown', rotateUp);
+
+// Rotate Left
+function rotateLeft(event) {
+  if (event.code !== 'ArrowLeft') {
+    return;
+  }
+  car.className = 'car transition left';
+}
+window.addEventListener('keydown', rotateLeft);
+
+// Rotate Right
+function rotateRight(event) {
+  if (event.code !== 'ArrowRight') {
+    return;
+  }
+  car.className = 'car transition right';
+}
+window.addEventListener('keydown', rotateRight);
```

Comment:
So these four functions will fire off everytime a key is pressed. It would be much more efficient to have one function handle your click events, and then have some logic to handle what key the user has pressed.

```
window.addEventListener('keydown', handleKeyPress)
function handleKeyPress(event) {}
```